cmd/alluxio/main.go

```diff
@@ -116,6 +123,41 @@ func handle() {
 		}
 	}))
 
+	if pprofAddr != "" {
+		glog.Infof("Enabling pprof with address %s", pprofAddr)
```

Comment:
Can we use the same log system in the controller. I suggest not to use `glog` here rahter than `github.com/go-logr/logr`.

Reply:
ok